application/views.py

```diff
@@ -1,7 +1,8 @@
 from flask import Blueprint, render_template, request, jsonify
 
 from application.auth.helpers import (current_user, authentication_required)
-from application.models import Team, Collection
+from application.models import Team, Collection, Request
+from application.helpers import (run_collection_checks)
```

Comment:
The brackets are redundant

Reply:
Why, because perhaps it's one item being imported